tests/openvino/native/quantization/test_weights_compression_statistics_caching.py

```diff
@@ -0,0 +1,160 @@
+    - Scale estimation
+    - Ignored scope
+    """
+    from nncf.openvino.statistics.aggregator import OVStatisticsAggregator
+
+    collect_statistics_spy = mocker.spy(OVStatisticsAggregator, "collect_statistics")
+    load_statistics_from_dir_spy = mocker.spy(OVStatisticsAggregator, "load_statistics_from_dir")
+    dump_statistics_spy = mocker.spy(OVStatisticsAggregator, "dump_statistics")
+
+    # Constant Parameters
+    subset_size = 4
+    mode = nncf.CompressWeightsMode.INT4_ASYM
+
+    # Compression Parameters
+    awq_values = [True, False]
+    group_size_values = [1, 4]
+    ratio_values = [0.4, 0.8]
+    sensitivity_metric_values = [
+        nncf.SensitivityMetric.HESSIAN_INPUT_ACTIVATION,
+        nncf.SensitivityMetric.MAX_ACTIVATION_VARIANCE,
+    ]
+    gptq_values = [False]  # [True, False] ticket: 155538
+    ignored_scope_values = [
+        IgnoredScope(),
+        IgnoredScope(
+            names=[
+                "__module.model.model.decoder.layers.1.self_attn.q_proj/prim::PythonOp/MatMul",
+                "__module.model.model.decoder.layers.11.self_attn.k_proj/prim::PythonOp/MatMul",
+                "__module.model.model.decoder.layers.11.self_attn.k_proj/prim::PythonOp/MatMul",
+            ]
+        ),
+    ]
+
+    MODEL_ID = "facebook/opt-125m"
+
+    dataset = datasets.load_dataset("wikitext", "wikitext-2-raw-v1", split="test")
+    tokenizer = AutoTokenizer.from_pretrained(MODEL_ID)
+    model = OVModelForCausalLM.from_pretrained(MODEL_ID, export=True, load_in_8bit=False, compile=False, stateful=False)
+
+    transform_fn = create_transform_fn(model, tokenizer)
+    quantization_dataset = nncf.Dataset(dataset, partial(transform_fn))
+
+    load_statistics_number = 0
+
+    # Test basic configurations (AWQ, group size, ratio, sensitivity metric, ignored_scope)
+    for awq, group_size, ratio, sensitivity_metric, ignored_scope in product(
+        awq_values, group_size_values, ratio_values, sensitivity_metric_values, ignored_scope_values
+    ):
+        print(
+            f"Testing configuration: awq={awq}, group_size={group_size}, ratio={ratio}, \
+            sensitivity_metric={sensitivity_metric}, ignored_scope is empty={len(ignored_scope.names) > 0}"
+        )
+
+        # Perform the compression test
+        nncf.compress_weights(
+            deepcopy(model.model),
+            mode=mode,
+            dataset=quantization_dataset,
+            ratio=ratio,
+            awq=awq,
+            gptq=False,
+            group_size=group_size,
+            scale_estimation=False,
+            subset_size=subset_size,
+            sensitivity_metric=sensitivity_metric,
+            ignored_scope=ignored_scope,
+            advanced_parameters=AdvancedCompressionParameters(statistics_dir_path=tmp_path / "statistics"),
+        )
+        load_statistics_number += 1
+
+    # Test advanced configurations (GPTQ, scale estimation)
+    for gptq, scale_estimation in product(gptq_values, [True, False]):
```

Comment:
I'd also verify `lora_correction` algorithm with the following advanced params to make it faster.
    adapter_rank: int = 8
    num_iterations: int = 0
    apply_regularization: bool = False
    subset_size: int = 4
    use_int8_adapters: bool = False

Reply:
Added lora_correction to test scope